backend/src/app.controller.ts

```diff
@@ -16,38 +17,30 @@ export class AppController {
   /**
    * Returns a list the ids of all nodes
    */
-  @Get('getAllNodes')
-  async getAllNodes(): Promise<number[]> {
-    return this.appService.getAllNodes();
+  @Post('queryAll')
+  async queryAll(@Body() query?: LimitQuery): Promise<QueryResult> {
```

Comment:
What's the purpose of this limit query? Can you specify the max number of nodes/edges that are returned?

Reply:
Yes, I forgot to create a comment on LimitQuery.